app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,7 @@
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>
   </head>
 
+  
```

Comment:
why the extra line? :O

Reply:
conflict in my file, because of the "/pages" vs the old "pages", when i resolve, probably accidentally added one line